run/send-push-message

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash -eu
 
-NUM_FILES_MOVED="$1"
-TITLE=${2:-"Dashcam Copy Complete"}
-MESSAGE="$NUM_FILES_MOVED file(s) were copied."
+TITLE="$1"
```

Comment:
Since all of the push messages start off with "TeslaUSB:", would it make sense to just prepend it here?

`TITLE="TeslaUSB: $1"`

`TITLE` would be the main action and `MESSAGE` can be the optional details (how many files were moved, skipped, failed).

Reply:
Valid Point. I have other things on the Pi (and elsewhere) that send a push notification using this utility, so I have used TeslaUSB: prefix to tell me which component is sending me the push notification. 

Ideally the push message should not be too wordy, so I have shortened the title to be only ```TeslaUSB:``` always so it makes even more sense to have it be a constant

Reply:
Thinking it through, I think that Title should uniquely identify the car if someone has more than 1 tesla so they know where the notification is coming from.